docs/general/server/transcoding.md

```diff
@@ -26,3 +23,7 @@ There are four types of playback; three of which involve transcoding. The type b
 - Remux: Changes the container but leaves both audio and video streams untouched.
 - Direct Stream: Transcodes audio but leaves original video untouched.
 - Transcode: Transcodes the video stream.
+
+## HDR to SDR Tone-mapping
+
+When the source video is in HDR, it will need to be tone-mapped to SDR when transcoding, as Jellyfin currently doesn't support HDR to HDR tone-mapping, or passing through HDR metadata. While this can be done in software, it is very slow and you may encounter situations where no modern consumer CPUs can transcode in real time. Therefore, a GPU is always recommended, where even a basic Intel iGPU can handle the load without problems.
```

Comment:
Not that ideal. We've received reports that N100 GPU is struggling on 4K60 dolby vision videos, but it is as fast as 8 core zen3 at dolby vision reshaping so still much more cost effective